idd/Energy+.idd.in

```diff
@@ -56166,23 +56262,293 @@ AirLoopHVAC:OutdoorAirSystem:EquipmentList,
        \reference AirLoopOAEquipmentLists
   A2 , \field Component 1 Object Type
        \required-field
+       \type choice
```

Comment:
@Nigusse Was there a reason that these were not done as \object-list validOAEquipmentTypes?  The IDD can use this approach even if we don't add another set of input processor validation.